### ISSUE TITLE
Sending a tip to non-existent publisher

### ISSUE THREAD

**Issue by gdregalo** [brave/browser-android-tabs#1389]
## Description 
The user can make a typo during typing a url, which results in "The site can't be reached" warning.
In this case on Rewards panel the publisher is presented as unverified and user can still send a tip to him. Sending a tip to non-existent publisher is confusing.  
There is case when the publisher exists but her host is inaccessible. 
For this case we might need to use DNS resolver to check if the url is valid and belongs to existing  publisher.

## Steps to reproduce 

   1. 
   2. 
   3. 

## Issue happens on 
   - Current Play Store version?
   - Beta build?

## Device details
   - Install type (ARM, x86):
   - Device (Phone, Tablet, Phablet):
   - Android version:
   
### Website problems only
- Does the issue resolve itself when disabling Brave Shields? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by NejcZdovc** [comment]
user can tip them, but no money will be actually send until that publisher is verified (user can only tip verified publishers). I think this is not a problem

**Comment by gdregalo** [comment]
@NejcZdovc Yes, user can only tip verified publishers. But we say "Tip has been sent..."  and it doesn't look professional because the publisher doesn't exist. The DNS resolver functionality is inbuilt into browser and  we can find out if the url points to a valid domain before registering the publisher in the database.
